.moban.yml

```diff
@@ -0,0 +1,48 @@
+overrides: coala.yaml
+
+name: coala
+contact: coala.analyzer@gmail.com
```

Comment:
no `@coala.io` address?

Reply:
see https://github.com/coala/coala/blob/master/setup.py#L88

Changing that is a separate issue (and probably an issue already exists, and possibly already a stale PR, or 5..).